t/plugin/authz-keycloak.t

```diff
@@ -337,8 +339,7 @@ passed
 GET /t
 --- response_body
 false
---- error_log
```

Comment:
I think we also need to check the error log?

Reply:
@liangliang4ward Please check this out.